erpnext/patches/v11_0/enable_include_exploded_items.py

```diff
@@ -0,0 +1,17 @@
+# Copyright (c) 2017, Frappe and Contributors
+# License: GNU General Public License v3. See license.txt
+
+from __future__ import unicode_literals
+import frappe
+
+def execute():
+	doctypes = {'Purchase Order': 'Buying', 'Purchase Invoice': 'Accounts', 'Purchase Receipt': 'Stock'}
+
+	for doctype, module in doctypes.items():
+		child_doc = doctype + ' Item'
+		frappe.reload_doc(frappe.scrub(module), 'doctype', frappe.scrub(child_doc))
+
+		frappe.db.sql(""" update `tab{child_doc}` as child_doc, `tab{parent_doc}` as parent_doc
```

Comment:
Why `include_exploded_items` should be set as 1 in existing transactions. Previously it was taking raw materials from BOM Item, so was not considering exploded items.

Reply:
Removed patch!